azure-pipelines-full.yml

```diff
@@ -30,6 +30,7 @@ jobs:
   pool:
     vmImage: 'macOS-10.14'
   steps:
+  - template: default-osx-template.yml
```

Comment:
Did you meant "- script: ./azure-pipeline-osx-setup.sh"?

Reply:
oh forgot to change that....